### PR TITLE
Remove identifier type

### LINKED ISSUE #155: Get rid of the Identifier Type for the PATH_ABANDON and PATH_STATUS frames
From PR #149 by @qdeconinck:

> We can of course delegate this on a subsequent PR, but I think we should get rid of the Identifier Type for the PATH_ABANDON and PATH_STATUS frames. Also, I wonder it the value 1 should rather be the default (given https://github.com/quicwg/multipath/issues/137).

### PR DESCRIPTION
**Pull request by mirjak**
fixes #155

### CODE REVIEW

File: draft-ietf-quic-multipath.md [hunk 1]
```diff
@@ -911,37 +911,12 @@ PATH_ABANDON frames are formatted as shown in {{fig-path-abandon-format}}.
...
-be empty.
-
-~~~
-  Path Identifier {
-    Identifier Type (i) = 0x00..0x02,
-    [Path Identifier Content (i)],
-  }
-~~~
-{: #fig-path-identifier-format title="Path Identifier Format"}
+Path Identifier:
```
Comment (mirjak) [inline draft-ietf-quic-multipath.md:914]:
```suggestion
Path Identifier:
```

File: draft-ietf-quic-multipath.md [hunk 2]
```diff
@@ -960,11 +935,9 @@ Reason Phrase:
 PATH_ABANDON frames SHOULD be acknowledged. If a packet containing
 a PATH_ABANDON frame is considered lost, the peer SHOULD repeat it.
 
-If the Identifier Type is 0x00 or 0x01, PATH_ABANDON frames MAY be sent
-on any path, not only the path identified by the Path Identifier Content
-field. If the Identifier Type if 0x02, the PATH_ABANDON frame MUST only
-be sent on the path that is intended to be abandoned.
-
+PATH_ABANDON frames MAY be sent
+on any path, not only the path identified by the Packet Number Space
```
Comment (mirjak) [inline draft-ietf-quic-multipath.md:939]:
```suggestion
on any path, not only the path identified by the Packet Number Space
```

File: draft-ietf-quic-multipath.md [hunk 3]
```diff
@@ -901,7 +901,7 @@ PATH_ABANDON frames are formatted as shown in {{fig-path-abandon-format}}.
 ~~~
   PATH_ABANDON Frame {
     Type (i) = TBD-02 (experiments use 0xbaba05),
-    Path Identifier (..),
+    Packet Number Space Identifier (i),
```
Comment (Yanmei-Liu) [inline draft-ietf-quic-multipath.md:904]: _Path Identifier (i),_

File: draft-ietf-quic-multipath.md [hunk 4]
```diff
@@ -976,7 +949,7 @@ PATH_STATUS frames are formatted as shown in {{fig-path-status-format}}.
 ~~~
   PATH_STATUS Frame {
     Type (i) = TBD-03 (experiments use 0xbaba06),
-    Path Identifier (..),
+    Packet Number Space Identifier (i),
```
Comment (Yanmei-Liu) [inline draft-ietf-quic-multipath.md:952]: _Path Identifier (i),_

**Review by Yanmei-Liu — COMMENTED** [review]
I agree that we could simplify the Path Identifier field and remove the type of it, but it's a little confusing using both Path Identifier and PNS Identifier in the PATH_STATUS / PATH_ABANDON Frame format.

I suggest using Path Identifier(i) in PATH_STATUS / PATH_ABANDON Frame, using Packet Number Space Identifier in ACK_MP and 1-RTT packets.

File: draft-ietf-quic-multipath.md (hunk 4 above)
Comment (mirjak) [reply draft-ietf-quic-multipath.md]: That was an oversight. Fixed now! Thanks!

**Review by Yanmei-Liu — APPROVED** [review]
Looks good to me

**Pull request merged**